chargeio/chargeio.csproj

```diff
@@ -0,0 +1,57 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <Version>1.1.0</Version>
```

Comment:
Version in readme does not match chargeio/chargeio.csproj (readme lists 1.0.0, chargeio.csproj has 1.1.0)